packages/cli/lib/validators/marketplaceValidators/theme/ModuleValidator.js

```diff
@@ -0,0 +1,129 @@
+const fs = require('fs');
+const BaseValidator = require('../BaseValidator');
+const { isModuleFolderChild } = require('@hubspot/cli-lib/modules');
+
+const MODULE_LIMIT = 50;
+
+class ModuleValidator extends BaseValidator {
+  constructor(options) {
+    super(options);
+
+    this.errors = {
```

Comment:
@KatzMitch I'm open to suggestions for better error messaging in all of these. I haven't gotten any design input on any of it, and I'm sure there's better phrasing that I could be using.

Reply:
Pulled in Amanda for feedback